test/system/irohad_test.cpp

```diff
@@ -26,7 +26,14 @@
 #include "common/types.hpp"
 #include "main/iroha_conf_loader.hpp"
 
+// workaround for redefining -WERROR problem
+#undef RAPIDJSON_HAS_STDSTRING
```

Comment:
what kind of error does it show?

Reply:
Warning about redefinition of RAPIDJSON_HAS_STDSTRING. Since we build tests with -WError build fails.